### PR TITLE
set zmon_scheduler_mem via CR

### PR DESCRIPTION
**Pull request by vetinari**
some clusters need an override

Signed-off-by: Hanno Hecker <hanno@zalando.de>

### CODE REVIEW

**Comment by mikkeloscar** [comment]
:+1:

**Comment by vetinari** [comment]
do we need extra permissions in the CR to set this for a specific cluster?

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -121,6 +121,7 @@ zmon_agent_replicas: "0"
 zmon_aws_agent_replicas: "0"
 zmon_redis_replicas: "0"
 zmon_scheduler_replicas: "0"
+zmon_scheduler_mem: "0Gi"
```
Comment (pitr) [inline cluster/config-defaults.yaml:124]: is this correct, zero gigs

Comment (mikkeloscar) [reply cluster/config-defaults.yaml:124]: This is for e2e tests. It doesn't really matter since replicas is set to `0` anyway

**Comment by gargravarr** [comment]
:+1:

**Pull request merged**